controllers/ldap.go

```diff
@@ -280,7 +280,6 @@ func (c *ApiController) SyncLdapUsers() {
 	}
 
 	c.ResponseOk(&LdapSyncResp{
-		Exist:  exist,
```

Comment:
why removed?
web/src/LdapSyncPage.js:49 has ref

Reply:
@theburyat

Reply:
removed because this field contains users, that are existed already when the sync has started, so they would not be updated, and we should show in ui. in my new flow, already existed users would be updated too, so that field in response is not needed. I fill make some fixes in front part in the next commit

Reply:
Now you don't want to show that user is exist? I think we should show it, but has ability to update anyway

Reply:
array of existed users contained users, that won't be synced, because they exist in casgate already, with sync we will show all users, synced and not synced